packages/kit/src/runtime/client/client.js

```diff
@@ -45,6 +45,9 @@ import { HttpError, Redirect, SvelteKitError } from '../control.js';
 import { INVALIDATED_PARAM, TRAILING_SLASH_PARAM, validate_depends } from '../shared.js';
 import { get_message, get_status } from '../../utils/error.js';
 import { writable } from 'svelte/store';
+import { VERSION } from 'svelte/compiler';
+
+const is_svelte_5 = VERSION.startsWith('5.');
```

Comment:
```suggestion
const is_svelte_5 = +VERSION.split('.')[0] >= 5;
```

Reply:
This will go away once we only support Svelte 5 in SvelteKit 3, which will be before Svelte 6, but better safe than sorry I guess..